### PR TITLE
Feature/capstone-conference-page

### PR DESCRIPTION
**Pull request by SanaTameem**
### Description 
I have created this Capstone project with the topic of Web Science using `HTML ` ` CSS ` and ` JS`.

- This is the link for Loom [Video Record of my project](https://www.loom.com/share/fdcc36fff39448038cdc52aa9c36d6df)

I have followed these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
- Colors.
- Typography
- Layout

I have built only these 2 pages:
- The home page.
- The about page.

Each of these pages have versions for 2 different screen sizes:
- Mobile: up to 768px wide.
- Desktop: 768px or wider.

Interactions
- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.

Mobile menu
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.

Dynamic page
- The section "Featured speakers" is created dynamically in JavaScript.
- I have used a JavaScript variable with the data about the speakers and used it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        <li class="menu-Item"><a href="#">Program</a></li>
+        <li class="menu-Item"><a href="#">Join</a></li>
+        <li class="menu-Item"><a href="#">Sponser</a></li>
+        <li class="menu-Item"><a href="#">News</a></li>
+        <li class="menu-Item"><a href="#">CC Campaign</a></li>
+      </ul>
+      <button class="close-btn">&times</button>
+    </div>
+    
+  </nav>
```
Comment (levy002) [inline index.html]: - [OPTIONAL] I highly recommend you wrap this navigation into the `header` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    </div>
+  </section>
+  <section class="footer-desktop">
+    <div class="footer-container">
+      <div class="footer-logo-container">
+        <a href="index.html"><img src="./Assets/websci_white_logo.png" alt="logo" class="footer-logo"></a>
+      </div>
+      <div class="footer-text">2021 ACM WEB SCIENCE.<br><br>Some rights deserved. All content on the website is licensed under a WebSci license. See <span><a href="https://webscience.org/privacy-policy/" class="privacy-link">https://webscience.org/privacy-policy/</a></span> for detailed privacy-policy conditions.</div>
+    </div>
+  </section>
```
Comment (levy002) [inline index.html]: - [ ] In order to follow HTML best practices as required, this footer section must be wrapped into the `footer` tag in order to meet the project's requirements.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+              <li class="menu-Item"><a href="#">Program</a></li>
+              <li class="menu-Item"><a href="#">Join</a></li>
+              <li class="menu-Item"><a href="#">Sponser</a></li>
+              <li class="menu-Item"><a href="#">News</a></li>
+              <li class="menu-Item"><a href="#">CC Campaign</a></li>
+            </ul>
+            <button class="close-btn">&times</button>
+          </div>
+         
+        </nav>
```
Comment (levy002) [inline about.html]: - [OPTIONAL] I highly recommend you wrap this navigation into the header tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+      </section>
+
+      <section class="footer-desktop-about">
+        <div class="footer-container">
+          <div class="footer-logo-container">
+            <a href="index.html"><img src="./Assets/websci_white_logo.png" alt="logo" class="footer-logo"></a>
+          </div>
+          <div class="footer-text">2021 ACM WEB SCIENCE.<br><br>Some rights deserved. All content on the website is licensed under a WebSci license. See <span><a href="https://webscience.org/privacy-policy/" class="privacy-link">https://webscience.org/privacy-policy/</a></span> for detailed privacy-policy conditions.</div>
+        </div>
+      </section>
```
Comment (levy002) [inline about.html]: - [ ] In order to follow HTML best practices as required, this footer section must be wrapped into the footer tag in order to meet the project's requirements.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+            <li class="nav-item"><a href="https://twitter.com/WebSciConf?ref_src=twsrc%5Etfw%7Ctwcamp%5Eembeddedtimeline%7Ctwterm%5Escreen-name%3AWebSciConf%7Ctwcon%5Es1_c1"><div class="twitter-img"><img src="./Assets/twitter_64px.png" alt="twitter" width="22px"></div></a></li>
+            <li class="nav-item"><a href="#">English</a></li>
+            <li class="nav-item"><a href="#">My Page</a></li>
+            <li class="nav-item"><a href="#">Logout</a></li>
+          </ul>
+          </div>
+          <div class="nav2-desktop">
+                <div class="logo"><a href="index.html"><img src="./Assets/orange-websci-logo.png" class="logo-img"></a></div>
+                <ul class="nav-list2">
+                  <li class="nav-item2"><a href="about.html">About</a></li>
```
Comment (levy002) [inline about.html]: - [ ] In order to increase the best User Experience(UX) on your website, I highly recommend you change the `About` navigation which navigates to the `About` page (which is the current or active page) to the `Home` which will navigate to `Home` whenever a user is on `About` page but want to go to the `Home` page. 

    - ### YOUR ABOUT PAGE NAVIGATION
    
    image

    - ### MY SUGGESTIONS ON ABOUT PAGE NAVIGATIONS
    image

File: scripts/about.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
+// mobile menu :
+// const homepage = document.querySelector('.homepage');
```
Comment (levy002) [inline scripts/about.js]: - [ ] Kindly make sure to remove the commented code in your codebase in order to keep it clean and professionally.

File: scripts/index.js [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+// featured title
+const featuredTitle = document.createElement('h2');
+featuredTitle.classList.add('speakers-title');
+featuredTitle.textContent = 'Featured Speakers';
+featuredSpeakers.appendChild(featuredTitle);
+
+// featured title line
+const featuredTitleLine = document.createElement('hr');
+// featuredTitleLine.src = './Assets/icons8-line-32.png';
```
Comment (levy002) [inline scripts/index.js]: - [ ] Kindly make sure to remove the commented code in your codebase in order to keep it clean and professionally.

File: scripts/index.js [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+
+speakers.forEach((speaker) => {
+  const htmlAdd = htmlOfSpeakers(
+    speaker.id,
+    speaker.name,
+    speaker.position,
+    speaker.intro,
+    speaker.photo,
+  );
+    // myWorksSection.insertAdjacentHTML('afterend',htmlAdd);
```
Comment (levy002) [inline scripts/index.js]: - [ ] Kindly make sure to remove the commented code in your codebase in order to keep it clean and professionally.

File: scripts/index.js [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+  document.querySelector('.speaker-6'),
+];
+moreSpeakers.addEventListener('click', () => {
+  arraySpeakers.forEach((speaker) => {
+    speaker.style.display = 'flex';
+  });
+});
+
+// mobile menu :
+// const homepage = document.querySelector('.homepage');
```
Comment (levy002) [inline scripts/index.js]: - [ ] Kindly make sure to remove the commented code in your codebase in order to keep it clean and professionally.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @SanaTameem   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Professional README :heavy_check_mark: 
- Professional PR :heavy_check_mark: 
- Good UI :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+        </div>
+      </section>
+
+      <section class="footer-desktop-about">
+        <div class="footer-container">
+          <div class="footer-logo-container">
+            <a href="index.html"><img src="./Assets/websci_white_logo.png" alt="logo" class="footer-logo"></a>
+          </div>
+          <div class="footer-text">2021 ACM WEB SCIENCE.<br><br>Some rights deserved. All content on the website is licensed under a WebSci license. See <span><a href="https://webscience.org/privacy-policy/" class="privacy-link">https://webscience.org/privacy-policy/</a></span> for detailed privacy-policy conditions.</div>
+        </div>
```
Comment (jerryowusu) [inline about.html]: As stated by the previous reviewer, kindly wrap this section in a footer tag for semantic and HTML best practices. 👍

File: about.html [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+              <div class="img-past2-overlay">
+                <div class="img-past-texts">
+                  <h2 class="img-past-year">2019</h2>
+                  <p class="img-past-info">Computer network and comunications</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        <!-- </div><img src="./Assets/websci2020.jpg"> -->
+        <!-- <img src="./Assets/websci2019.jpg"> -->
```
Comment (jerryowusu) [inline about.html]: [Optional] Kindly consider removing this commented code from your codebase in order to keep it clean and professional. 😉

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @SanaTameem ,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close


 ## Highlights
Great effort in making some of the requested changes :+1: 
No linter errors :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+      </section>
+    <section>
+      <section class="footer-mobile">
+        <div class="footer-container">
+          <div class="footer-logo-container">
+            <img src="./Assets/websci_white_logo.png" alt="logo" class="footer-logo">
+          </div>
+          <div class="footer-text">2021 ACM WEB SCIENCE. <br> Some rights deserved.</div>
+        </div>
+      </section>
```
Comment (franclobo) [inline about.html]: - **[OPTIONAL]** As you said in the class this is a footer so please 🙏🏽  use the footer tag for the best practices of HTML & CSS.

**Review by franclobo — APPROVED** [review]
Hi @SanaTameem, 

# STATUS: APPROVED 😃 

git-merge

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Good topic 💯 

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**